docs/guides/Checkout/customization/displaying-sla-by-item-in-checkout.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Displaying SLA by item in Checkout"
+slug: "displaying-sla-by-item-in-checkout"
+hidden: false
+createdAt: ""
+updatedAt: ""
+excerpt: "This guide describes how to display individual shipping estimates for items in a customer's cart on an ecommerce platform using the Checkout API."
+hidePaginationPrevious: false
+hidePaginationNext: false
+---
+
+To enhance the customer experience, some stores may want to display precise shipping estimates for each item in the customer's cart. This can be particularly useful for ecommerce platforms that have sellers with different shipping capabilities and delivery times. This guide will walk through the process of displaying the Service Level Agreement (SLA) by item using the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api).
+
+<CH.Scrollycoding>
+
+## Enable the `useIndividualShippingEstimates` flag
+
+First, it is necessary to enable the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api) behavior to fill the `shippingEstimateDate` with the SLA information. To do so, you must update the store's `orderForm` with the [Update orderForm configuration](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pvt/configuration/orderForm) endpoint. You must include the flag `useIndividualShippingEstimates` to `true` as in the example shown.
```

Comment:
"First, you need to enable the..."

"You must set the `useIndividualShippingEstimates` flag to `true` as in the example below.